temp_utility_scripts/remove_incomplete_attachments.py

```diff
@@ -118,6 +130,41 @@ def remove_unattached_image_paths(self):
         self.logger.warning(f"Number of attachments/collectionobjectattachments deleted: {len(attachment_id_test_count)}")
 
 
+    def remove_attachements_no_record(self):
```

Comment:
I think this just removes attachments - it doesn't check if the record is present or absent, yes? if so, a rename is in order; it's just "remove attachments".